tests/async-loader-with-long-load.html

```diff
@@ -0,0 +1,60 @@
+<!doctype html>
+<!--
+@license
+Copyright (c) 2018 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+<html>
+
+<head>
+  <title>Test async loader with long sync load</title>
+  <meta charset="UTF-8">
+  <script src="./wct-config.js"></script>
+  <script>window.callback = null; WCT.waitFor = (cb) => { window.callback = cb; }</script>
+  <script src="../node_modules/wct-browser-legacy/browser.js"></script>
+
+  <script async src="../webcomponents-loader.js"></script>
+  <script>
+    const start = Date.now();
+    let diff = 0;
+    while (diff < 5000) {
```

Comment:
Maybe add a clarifying comment here. Something like `we mimic a long loading script to make sure the document readystate is loading when the loader executes.`

Reply:
Added a detailed explanation in the comments